src/types/algoliasearch.ts

```diff
@@ -39,6 +40,8 @@ export type SearchClient = {
   search: DefaultSearchClient['search'];
   searchForFacetValues: DefaultSearchClient['searchForFacetValues'];
   addAlgoliaAgent?: DefaultSearchClient['addAlgoliaAgent'];
+  initIndex?: DefaultSearchClient['initIndex'];
+  transporter?: SearchClientV4['transporter'];
```

Comment:
what do we use transporter for?

Reply:
It's for

```
         if (instantSearchInstance.client.transporter) {
          instantSearchInstance.client.transporter.userAgent.value =
            'answers-test';
        } 
```

Reply:
I think we maybe should instead @ts-ignore there, since the change there is only while it's experimental